gateway/plugin/external.go

```diff
@@ -154,3 +146,20 @@ func (s ExternalServiceQuery) SetReplicas(serviceName string, count uint64) erro
 
 	return err
 }
+
+// This methods tries to parse the provided raw label value and if it fails
+// it will return the provided fallback value and log and message
+func extractLabelValue(rawLabelValue string, fallback uint64 ) uint64{
```

Comment:
Looks like a good refactor.

Comments on functions (when given) should start with the name of the function i.e. `extractLabelValue will parse the provided raw label...`

Reply:
Okay, that was something I didn't know. I will fix that then.